test/unit/downloader.test.ts

```diff
@@ -171,14 +171,41 @@ _test('Downloader class with optimisation', async (t) => {
     const httpOrHttpsRemoved = downloader.stripHttpFromUrl(testImage);
```

Comment:
Please move `stripHttpFromUrl()` to a a "utils' js module so we can use it as a function. Having it as a method of the Downloader is not proper.